client/src/components/Login/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { Modal, Button } from 'antd';
+import Login from './Login';
+import logo from '../../assets/Nova Logo.png';
+import './style.css';
+
+function LoginModal() {
```

Comment:
Convert all the functions to the arrow function.
If you edit the file's name so they match the functions components, I mean the Login file is actually the loginModal component.
And I think the components name is confusing, the modal is called just Login and the Login btn is called LoginModal